test/inductor/test_flex_attention.py

```diff
@@ -90,6 +94,18 @@ def temp_float32_matmul_precision(precision: str):
         torch.set_float32_matmul_precision(original_precision)
 
 
+def skip_on_cpu(test_func):
+    """Decorator to skip tests that are not supported on CPU."""
+    decorated_func = skipCPUIf(True, "Not supported on CUDA")(test_func)
```

Comment:
```suggestion
    decorated_func = skipCPUIf(True, "Not supported on CPU")(test_func)
```

Reply:
There are alot of flaky test failures so will land in follow up, but goood catch